BookingsApi/BookingsApi/Controllers/JudiciaryPersonController.cs

```diff
@@ -159,11 +167,18 @@ public async Task<IActionResult> BulkJudiciaryLeaversAsync(IEnumerable<Judiciary
         [ProducesResponseType((int)HttpStatusCode.BadRequest)]
         public async Task<IActionResult> PostJudiciaryPersonBySearchTerm(SearchTermRequest term)
         {
-            var query = new GetJudiciaryPersonBySearchTermQuery(term.Term);
-            var personList = await _queryHandler.Handle<GetJudiciaryPersonBySearchTermQuery, List<JudiciaryPerson>>(query);
-            var mapper = new JudiciaryPersonToResponseMapper();
-            var response = personList.Select(x => mapper.MapJudiciaryPersonToResponse(x)).OrderBy(o => o.Username).ToList();
-            return Ok(response);
+            if (_flagsService.GetFeatureFlags().EJudFeature)
```

Comment:
[GENERAL-COMMENT] Going forward consider if there might be a need for a _flagsService.GetFeatureFlag(flagName);